src/jvm/test/org/jetbrains/kotlinx/lincheck_test/verifier/linearizability/RendezvousChannelCustomTest.kt

```diff
@@ -41,7 +41,7 @@ class RendezvousChannelCustomTest : VerifierState() {
     private val pollFun = RendezvousChannelCustomTest::poll
 
     @Test
-    fun testCancellation_01() = withLincheckJavaAgent(InstrumentationMode.STRESS) {
```

Comment:
Shouldn't the verifier track the `getResult` calls?

Reply:
Yes, but `withLincheckJavaAgent` is already called from the `verify` method.

Reply:
Is it? I don't see that. 

By the way, I also think we currently might have a bug with the model-checking mode, as the lazy transformation might not work in the verification phase, as we do not analyze anything in this phase, and, therefore, we do not ensure that the code dependencies are transformed.

Reply:
> Is it?

Yes, here I mean special `verify` method that we only use in `RendezvousChannelCustomTest` and similar tests that test the verifier algorithm. This method calls `withLincheckJavaAgent`, so everything is okay. 

> By the way, I also think we currently might have a bug with the model-checking mode, as the lazy transformation might not work in the verification phase, as we do not analyze anything in this phase, and, therefore, we do not ensure that the code dependencies are transformed.

Yes, maybe, but that is a separate issue.

Reply:
Could you write a test for this and check whether there is such an issue? It does not prevent the current PR from being merged, of course.